backends/nft/register.go

```diff
@@ -25,7 +27,12 @@ func (b *backend) Sink() localsink.Sink {
 	sink := fullstate.New(&b.cfg)
 
 	PreRun()
-	sink.Callback = Callback
+
+	ct := conntrack.New()
+	sink.Callback = fullstatepipe.New(fullstatepipe.ParallelSendSequenceClose,
```

Comment:
I think we should be calling the tasks in serial order 1)Apply the rules 2)Delete the stale endpoints.